module-4/backend/week_two.md

```diff
@@ -1,22 +1,46 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's one difference between ES5 and ES6?
+  One difference between ES5 and ES6
+  is the import and export way of requiring files.
 
-Note: When you're done, submit a PR. 
+2. What's the difference between asynchronous and synchronous JavaScript?
+  Asynchronous JavaScript does not wait for a line of code to get executed before running the next one while synchronous JavaScript runs a line at a time.
```

Comment:
Yes - make sure you're clear that JS is a synchronous language, but some functions behave in an async manner.